app/templates/_server.js

```diff
@@ -0,0 +1,16 @@
+var express = require('express');
+var app = express();
+var path = require('path');
+
+app.use('/static', express.static('public'));
+app.get('/', function (req, res) {
+	res.sendFile(path.resolve(__dirname, 'public/index.html'));
+});
+
+var server = app.listen(3000, function () {
+	var host = server.address().address;
+	var port = server.address().port;
+
+	console.log('<%= newAppDir %> listening at http://%s:%s', host, port);
```

Comment:
nice! 👍 it is definitely faster then looking at the `package.json` and getting the name of the project.